.circleci/config.yml

```diff
@@ -16,7 +29,32 @@ jobs:
       - run:
           name: Verify
           command: make ci
+      - run:
+          name: Build collector 
+          command: make otelcontribcol
       - store_artifacts:
           path: testbed/tests/results
       - store_test_results:
           path: testbed/tests/results/junit
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+  publish:
+    docker:
+      - image: circleci/golang:1.12
+    steps:
+      - attach_workspace:
+          at: .
+      - setup_remote_docker
+      - run:
+          name: Build image
+          command: |
+            make docker-otelcontribcol
+            docker tag otelcontribcol:latest omnition/opentelemetry-collector-contrib:${CIRCLE_TAG}
```

Comment:
What will be the value of ${CIRCLE_TAG}

Reply:
Git tag when the build is started by a tag and this step only executes when the tag matches the regex specified at top of the file. So this should always be something like `vX.Y.Z`

Reply:
Hmm, just realized that we don't want to publish the `v` prefix to docker. Will update to take care of that.

Reply:
Fixed.